cypress.config.js

```diff
@@ -9,6 +9,7 @@ export default defineConfig({
   e2e: {
     specPattern: 'cypress/**/*.{cy,spec}.{js,jsx,ts,tsx}',
     defaultCommandTimeout: 60000,
+    chromeWebSecurity: false,
```

Comment:
adicionar isso apenas no describe do teste de billing, evitando deixar inativo para os demais teste, pois é requerido apenas em bililng

Reply:
o Cypress.config() pode ser usado nos hooks do cypress, before all e after all para essa alteração, segue o link:
https://docs.cypress.io/api/cypress-api/config

Reply:
https://docs.cypress.io/guides/guides/web-security#Set-chromeWebSecurity-to-false
CypressError: Cypress.config() can never override chromeWebSecurity because it is a read-only configuration option.

a config de chromeWebSecurity só pode ser alterada dentro do arquivo de config do cypress, ao tentar alterar utilizando  o cypress.config() apresenta o erro a cima